File: app/src/main/java/com/example/mitroshin/pagingdemo/model/entity/UserLinks.java
package com.example.mitroshin.pagingdemo.model.entity;

import com.google.gson.annotations.SerializedName;

public class UserLinks {

    @SerializedName("self")
    public final String self;

    @SerializedName("html")
    public final String html;

    @SerializedName("photos")
    public final String photos;

    @SerializedName("likes")
    public final String likes;

    @SerializedName("portfolio")
    public final String portfolio;

    public UserLinks(String self,
                     String html,
                     String photos,
                     String likes,
                     String portfolio) {
        this.self = self;
        this.html = html;
        this.photos = photos;
        this.likes = likes;
        this.portfolio = portfolio;
    }

    @Override
    public String toString() {
        return "UserLinks{" +
                "self='" + self + '\'' +
                ", html='" + html + '\'' +
                ", photos='" + photos + '\'' +
                ", likes='" + likes + '\'' +
                ", portfolio='" + portfolio + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/mitroshin/pagingdemo/model/entity/UserLinks.kt
package com.example.mitroshin.pagingdemo.model.entity

import com.google.gson.annotations.SerializedName

data class UserLinks(
        @field:SerializedName("self")
        val self: String,

        @field:SerializedName("html")
        val html: String,

        @field:SerializedName("photos")
        val photos: String,

        @field:SerializedName("likes")
        val likes: String,

        @field:SerializedName("portfolio")
        val portfolio: String)
